webapp-django/crashstats/crashstats/views.py

```diff
@@ -1118,6 +1119,22 @@ def crashes_per_day(request, default_context=None):
     context['graph_data'] = graph_data
     context['report'] = 'daily'
 
+    errors = []
+    for error in results.get('errors', []):
+        if not error['type'] == 'shards':
+            continue
+
+        week = int(error['index'][-2:])
+        year = int(error['index'][-6:-2])
+        day = isoweek.Week(year, week).monday()
+        percent = error['shards_count'] * 100 / 5
```

Comment:
Does the following look right to you for what you're expecting? 

```
>>> 1 * 100 / 5
20
>>> 2 * 100 / 5
40
>>> 3 * 100 / 5
60
>>> 5 * 100 / 5
100
```

Where's the 5 come from? That seems like a magic number.